pom.xml

```diff
@@ -16,7 +23,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <cloudtrust.version>${project.version}</cloudtrust.version>
+        <cloudtrust.version>3.0.0</cloudtrust.version>
```

Comment:
No... `${project.version}` was fine.
If you are developping a 3.0.0-SNAPSHOT, you can assume that 3.0.0 does not exists.
Usually, you will try to hard code the less version as possible.

Reply:
good catch

Reply:
Crossing version numbers is really a very bad idea

Reply:
It seems we all agree it is not a good practice but it seems we do not find a way to make it works with SNAPSHOT version number. 
It seems it make non sense to invest more time on this point which should a very limited impact and could easily be improved later.

Reply:
It seems @basbeu can't make it clean in many places... I think it is because we don't use -SNAPSHOT everywhere. If you always use crossed version numbers, you often think your are pointing to a given version but are using another one.